.github/workflows/ci-test.yml

```diff
@@ -235,11 +235,12 @@ jobs:
           fi
           if [[ $num_fails != 0 ]]; then
             echo -e "$num_fails test(s) failed:\n${failed_tests}"
+          else
+            echo -e "No issues detected in this build". > ${MESSAGE_PAYLOAD_FILE}
```

Comment:
I think we want the flaky e2e test push alert to be "no news is good news".  If we go with this, it would create lots of noise in the channel because the if-on-failure condition (line 242) is removed below. 

It seems too good to be true that there was not a single flaky e2e test in the last ~4 weeks.  I walk through the retry logic above, the `${FLAKY_TESTS_FILE}` shouldn't be empty if there is a flaky test.  The only thing I can think of is the > comparator used in line 227 that would somehow act as string instead of integer comparator, which would then cause the condition to fall through the if-elif block.  

Got an idea to try - I supposed the GHA container runs in bash, in this case, we can try `-gt` instead of `>`, ie `[[ $retry -gt 1 ]]`.

Reply:
good theory, but the last job to report an error had that line in it's current form:

The slack message
https://toroeng.slack.com/archives/C011SK60RJ8/p1611701016000100

The workflow
https://github.com/diem/diem/actions/runs/513440085/workflow#L226

Reply:
I see, that's the only flaky detection after the switch to GHA. It could be the e2e tests becoming stable or the detection/alert not reporting as it should. I'd like to think the former :)

Reply:
yeah, but too good to be true.   extraordinary claims, require extraordinary evidence, hence I want to see this push to flaky-tests for a month or so.

Reply:
Back to the first point, if we go with the change here, this job will always send push alerts to the channel regardless of the state of tests.  Is this a temporary so as to debug (ie we will disable after some time) ?

Reply:
That's the plan.   I just want to see it....  I've not seen an e2e test failure for a long time, but I just don't _trust_ it.